kmmresult/src/commonTest/kotlin/KmmResultTest.kt

```diff
@@ -216,4 +216,21 @@ class KmmResultTest {
             result.transform { catching { fn(it) }}
         }
     }
+
+    @Test
+    fun testNonFatal() {
+
+        runCatching { throw CancellationException() }
```

Comment:
is this testing whether `catchingUnwrapped` is still necessary, or why is this here?